lms/envs/bok_choy.env.json

```diff
@@ -79,7 +79,6 @@
         "ENABLE_INSTRUCTOR_ANALYTICS": true,
         "ENABLE_S3_GRADE_DOWNLOADS": true,
         "ENABLE_THIRD_PARTY_AUTH": true,
-        "ENABLE_COMBINED_LOGIN_REGISTRATION": true,
```

Comment:
No. Since the default in `envs/common.py` would now be `true`, there's no need to override the default and set it true in the test file, so I removed this. But there'd be no harm in leaving it in, either.